docs/individual_sheets.md

```diff
@@ -220,47 +223,53 @@ options for each row are clickable links in the chart below:
     </td>
     <td class="row-4">
       You can put either: 
-      <em style="font-weight: bold">TRUE</em>, <em style="font-weight: bold">FALSE</em>, or <em>you can leave blank for False</em>
+      <em>project name</em> or <em style="font-weight: bold">FALSE</em>
     </td>
     <td class="row-4">
       <em>Your notes</em>
     </td>
   </tr>
 </table>
 
-??? note "use_existing"
-    `use_existing` column field is best for when you already have a material within your project and 
-    you just want to name it and use it within your Excel file instead of creating a new material on every upload 
-
+<details>
```

Comment:
great use of regular HTML to make the documentation less ties to a specific software and be more compatible overall!